go/vt/vttablet/tabletserver/query_engine_test.go

```diff
@@ -192,14 +192,14 @@ func TestQueryPlanCache(t *testing.T) {
 	} else {
 		qe.SetQueryPlanCacheCap(1)
 	}
-	firstPlan, err := qe.GetPlan(ctx, logStats, firstQuery, false, 0, nil)
+	firstPlan, err := qe.GetPlan(logStats, firstQuery, false, 0, nil)
 	if err != nil {
 		t.Fatal(err)
 	}
 	if firstPlan == nil {
 		t.Fatalf("plan should not be nil")
 	}
-	secondPlan, err := qe.GetPlan(ctx, logStats, secondQuery, false, 0, nil)
+	secondPlan, err := qe.GetPlan(logStats, secondQuery, false, 0, nil)
```

Comment:
This test is failing for me with the following error.
```
--- FAIL: TestQueryPlanCache (0.00s)
    query_engine_test.go:169: expected query plan cache to contain 1 entries, found 2
```
`firstQuery` and `secondQuery` are not the same, so I would expect the plan cache to contain both. Why are we expecting only one?